src/components/action/action.tsx

```diff
@@ -165,6 +183,21 @@ export class Action implements InteractiveComponent, LoadableComponent {
     ) : null;
   }
 
+  renderIndicatorText(): VNode {
+    const { indicator, intlIndicator, indicatorMessage, indicatorId, buttonId } = this;
+    return (
+      <div
+        aria-labelledby={buttonId}
+        aria-live="polite"
+        class={CSS.indicatorText}
+        id={indicatorId}
+        role="region"
+      >
+        {indicator ? indicatorMessage ?? intlIndicator : null}
```

Comment:
When the `indicatorMessage` prop is provided with a string, this works really well. However, no prop present provides no context. Would it be possible to have a default value for the indicator such as "indicator present for ${text}"?

Reply:
@anveshmekala will have to add a string for this and update it in his PR.